helmfile/charts/prometheus-alerts/templates/alerts/daily-checks.yaml

```diff
@@ -19,13 +19,24 @@ spec:
     rules:
     - alert: S3BucketContentOver{{.Values.s3BucketPercentLimit}}Percent
       annotations:
-        description: Over {{.Values.s3BucketPercentLimit}} percent of the total s3bucket size limit is used.
-        summary: Over {{.Values.s3BucketPercentLimit}} percent of the total s3bucket size limit is used.
+        message: Over {{.Values.s3BucketPercentLimit}} percent of the total s3bucket size limit is used in bucket {{`{{ $labels.bucket }}`}}.
       expr: |-
         (
-          sum(max_over_time(s3_objects_size_sum_bytes[1h])) >= ({{ .Values.s3BucketSizeQuotaGB }} * (1024^3)) * ({{ .Values.s3BucketPercentLimit }} / 100)
+          sum(max_over_time(s3_objects_size_sum_bytes[1h])) by (bucket) >= ({{ .Values.s3BucketSizeQuotaGB }} * (1024^3)) * ({{ .Values.s3BucketPercentLimit }} / 100)
         )
       for: 1h
       labels:
         severity: warning
+{{- if .Values.s3BucketNrOfObjectsAlarm }}
+    - alert: S3BucketOverNrOfObjects
+      annotations:
+        message: Over 1638400 objects are present in the s3-bucket {{`{{ $labels.bucket }}`}}.
+      expr: |-
+        (
+          sum(max_over_time(s3_objects[2h])) by (bucket) > 1638400
```

Comment:
Where is the 1638400 taken from?
Shouldn't it be configurable?
Also if the limit is x, shouldn't the alert fire if the number of objects is close to being hit, e.g. y*x < x, where y < 1.

Reply:
As the number '1638400' is Cleura exclusive, me and @Xartos thought it could make sense to just make it a "false" or "true" condition but its by no means finale. We will think about it today again! :+1:

Reply:
Safespring sto1 also have a default object limit. It's 10^6 if I'm not mistaken

Reply:
Now its done with a percentage and so on! :+1:

Reply:
to come: a way to exlude buckets we dont want to include into this.